ros2run/ros2run/command/run.py

```diff
@@ -56,4 +60,5 @@ def main(self, *, parser, args):
             raise RuntimeError(msg)
         if path is None:
             return 'No executable found'
-        return run_executable(path=path, argv=args.argv)
+        prefix=args.prefix[0] if args.prefix is not None else ''
```

Comment:
nitpick: PEP8 needs whitespace around the `=` (this will also cause our tests to fail)